src/i18n/locales/en.json

```diff
@@ -15,6 +15,11 @@
   "components.downloadTable.header.ltsStarts": "Active LTS Start",
   "components.downloadTable.header.maintenanceStart": "Maintenance LTS Start",
   "components.downloadTable.header.endOfLife": "End of Life",
+  "components.downloadTable.status.endOfLife": "End of Life",
```

Comment:
Please remove the translation and put it in English. The respective team will do a pr for that.

Reply:
@AugustinMauroy : Updated in the latest commit.